src/components/app.jsx

```diff
@@ -47,6 +47,12 @@ const App = React.createClass({
               </div>
             </section>
             {this.props.children}
+            <footer style={{marginTop: "5em"}}>
+              <hr/>
+              <a href={config.gitWebpageUrl} rel="external" target="_blank">
```

Comment:
No, I don't have any source and it seems artificial to add it :)

I kept `target="_blank"`.